lib/node_modules/@stdlib/number/float64/to-json/benchmark/benchmark.js

```diff
@@ -0,0 +1,95 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var PINF = require( '@stdlib/constants/float64/pinf' );
+var NINF = require( '@stdlib/constants/float64/ninf' );
+var pkg = require( './../package.json' ).name;
+var toJson = require( './../lib' );
+
+
+// MAIN //
+
+bench( pkg, function benchmark( x ) {
+	var v;
+	var i;
+
+	x.tic();
+	for ( i = 0; i < x.iterations; i++ ) {
+		v = toJson( NaN );
```

Comment:
Instead of testing multiple values like this, only test one at a time. See benchmarks which use the `values[ i%values.length ]` pattern.